### PR TITLE
Utils/logging

### PR DESCRIPTION
**Pull request by ZeyadTamimi**

### CODE REVIEW

File: kaos/__init__.py [hunk 1]
```diff
@@ -11,14 +18,38 @@
...
+    # Configure Logging
+    logging_level = app.config.get('LOGGING_LEVEL', 'INFO').upper()
+    logging_directory = app.config.get('LOGGING_DIRECTORY', '.')
+    logging_file = app.config['LOGGING_FILE_NAME']
+
+    numeric_level = getattr(logging, logging_level, None)
+    if not isinstance(numeric_level, int):
+        raise ValueError('Invalid log level: {}'.format(logging_level))
+
+    # Create the logging directory if it doesnt exist
```
Comment (lsav) [inline kaos/__init__.py]: doesn't

File: kaos/settings.cfg [hunk 2]
```diff
@@ -9,6 +9,14 @@ DATABASE_SETTINGS = {
     'password': 'kaosuserpassword',
 }
 
+CALCULATION_PRECISION = 100
+
+LOGGING_LEVEL = 'DEBUG'
+LOGGING_FILE_NAME = 'kaos_unittest_log_%Y_%m_%d_%H_%M_%S'
```
Comment (lsav) [inline kaos/settings.cfg]: This should probably have a different name since it'll be the "production" log

File: kaos/models/models.py [hunk 3]
```diff
@@ -33,7 +33,7 @@ class Satellite(SavableModel, DB.Model):
     platform_name = DB.Column(DB.String(50), nullable=False)
     orbit_segments = DB.relationship("OrbitSegment", backref='satellite', lazy=True)
     orbit_records = DB.relationship("OrbitRecord", backref='satellite', lazy=True)
-    maximum_altitude = DB.Column(DB.Float, nullable=True)
+    maximum_altitude = DB.Column(DB.Float)
```
Comment (lsav) [inline kaos/models/models.py]: I'm a little confused by this. In models/parser.py:67-69, we create the satellite as follows:

```
    sat = Satellite(platform_name=os.path.splitext(os.path.basename(filename))[0])
    sat.save()
    DB.session.commit()
```

The `maximum_altitude` is not specified until later. Shouldn't making it non-nullable cause an error with our current parser?

Comment (ZeyadTamimi) [reply kaos/models/models.py]: Nope, when you create a value it still gets a default value ( I believe 0 )

**Pull request merged**